packages/test_express/package.json

```diff
@@ -14,18 +14,8 @@
   "directories": {
     "test": "test"
   },
-  "devDependencies": {
-    "aws-sdk": "^2.304.0",
-    "aws-xray-sdk-core": "3.2.0",
-    "aws-xray-sdk-express": "3.2.0",
-    "chai": "^3.5.0",
-    "cls-hooked": "^4.2.2",
-    "eslint": "^4.18.2",
-    "express": "^4.16.4",
-    "mocha": "^3.5.3"
-  },
   "scripts": {
-    "test": "mocha --recursive ./test/ -R spec"
+    "test": "mocha --recursive --exit ./test/ -R spec"
```

Comment:
Not sure why is the `--exit` flag used here and not in other package?

Reply:
Great question. This package has tests that spin up localhost servers (like WireMock), and even though the servers are closed at the end of all the unit tests (at least as far as I can tell) they still caused the test execution to hang. The `--exit` flag forces the test execution to exit once all the unit tests are complete.

The reason this was only a problem in this PR is because `mocha`, the testing framework, had this `--exit` flag as the default behavior in the version we were using before, then when we upgraded in this PR and the behavior became opt in. We can include the `--exit` flag in all the packages for consistency, what do you think?

Reply:
Ah I see! Well, according to the mocha [doc](https://mochajs.org/#-exit) the use of `--exit` flag is required only if the tests hang and is not really recommended. If the tests in other packages do not hang, I don't see a necessity to add the `--exit` flag.